compute/k8s-services/main.tf

```diff
@@ -582,6 +582,27 @@ module "blackbox_exporter_flux_manifests" {
   depends_on = [module.monitoring_kube_prometheus_stack, module.platform_fluxcd]
 }
 
+# --------------------------------------------------
+# helm Exporter
+# --------------------------------------------------
+
+module "helm_exporter_flux_manifests" {
+  source             = "../../_sub/monitoring/helm-exporter"
+  count              = var.helm_exporter_deploy ? 1 : 0
+  cluster_name       = var.eks_cluster_name
+  helm_chart_version = var.helm_exporter_helm_chart_version
+  github_owner       = var.helm_exporter_github_owner
```

Comment:
This could be rewritten to:

```
  github_owner       = var.helm_exporter_github_owner != null ? var.helm_exporter_github_owner : var.platform_fluxcd_github_owner
  repo_name          = var.helm_exporter_repo_name != null ? var.helm_exporter_repo_name : var.platform_fluxcd_repo_name
  repo_branch        = var.helm_exporter_repo_branch != null ? var.helm_exporter_repo_branch : var.platform_fluxcd_repo_branch
```

This means that if the three variables are not defined, it will use the similar variables for Flux in general.

Reply:
Okie dokie, you will have to explain to me why we wouldn't define these in the .hcl but i have added them.